README.md

```diff
@@ -3,21 +3,132 @@
+
+- you want to be prompted for only the settings necessary to launch MLSpace with minimal configuration changes; or...
+- you want to be prompted only for the necessary settings as well as values that are commonly modified (VPC configuration, IAM roles, etc)
+- you want to configure and deploy MLSpace with a generated config file which is not committed to git
+
+After running the MLSpace Config Wizard, a new file will be generated: `/lib/config.json`.
+
+When MLSpace is deployed it will merge the settings in `config.json` and `constants.ts` to determine the final configuration settings (giving precedence to values set in `config.json`).
+
+Any values left empty while using the MLSpace Config Wizard will default to what is set in the `lib/constants.ts` file.
+
+The MLSpace Config Wizard can be invoked with the command:
+
+```Bash
+npm run config
+```
+
+This will prompt you to choose between Basic Config and Advanced Config.
+
+- Basic Config - only prompts for the properties which must be set in order to deploy MLSpace.
+- Advanced Config - prompts for required fields as well as optional configurations which are commonly customized.
+
+If selecting Basic Config, the properties you will be prompted for are:
+
+- AWS account ID: the AWS account ID for the account MLSpace will be deployed into
+- AWS region: the region that MLSpace resources will be deployed into
+- OIDC URL: the OIDC endpoint that will be used for MLSpace authentication
+- OIDC Client Name: the OIDC client name that should be used by MLSpace for authentication
+
+If selecting Advanced Config you will be prompted for the same properties Basic Config prompts for, as well as other optional values. Anything not specific will use the defaults in `constants.ts` and/or provisioned by MLSpace. 
+
+The Advanced Config will ask:
+
+**Do you want to use existing VPC?**
+If you answered yes you will be prompted for:
+
+- VPC Name: if MLSpace is being deployed into an existing VPC this should be the name of that VPC
+- VPC ID: if MLSpace is being deployed into an existing VPC this should be the ID of that VPC
+- VPC Default Security Group: if MLSpace is being deployed into an existing VPC this should be the default security group of that VPC
+
+**Do you want to use existing IAM Roles?**
+If you answered yes you will be prompted for:
+
+- S3 Reader Role ARN: arn of an existing IAM role to use for reading from the static website S3 bucket
+- Bucket Deployment Role ARN: arn of an existing IAM role to use for deploying to the static website S3 bucket
+- Notebook Role ARN: arn of an existing IAM role to associate with all notebooks created in MLSpace
+- App Role ARN: arn of an existing IAM role to use for executing the MLSpace lambdas
+- EMR Default Role ARN: arn of an existing IAM role that will be used as the 'ServiceRole' for all EMR clusters
+- EMR EC2 Instance Role ARN: arn of an existing role that will be used as the 'JobFlowRole' and 'AutoScalingRole' for all EMR clusters
```

Comment:
Minor quip: Shouldn't ARN always be capitalized since it is an initialism?

Reply:
Wouldn't hurt; it seems inconsistent in our README and install.md so I'll update it wherever I see it